src/index.ts

```diff
@@ -224,6 +249,37 @@ export class AthenaNodeClient {
         return query.results;
     }
 
+
+    /**
+     * Get statistics from a query execution
+     *
+     * @private
+     * @template T
+     *
+     *
+     * @returns {Promise<T[]>} - parsed query result rows
+     * @memberof AthenaNodeClient
+     * @param {string} executionId
+     */
+    private async getQueryStatistics(executionId: string): Promise<Statistics> {
+        const input: GetQueryExecutionCommand = new GetQueryExecutionCommand({
+            QueryExecutionId: executionId
+        });
+
+        const response = await this._client.send(input);
+        const bytes = response?.QueryExecution?.Statistics?.DataScannedInBytes ?? null;
```

Comment:
Yo lo dejaría así que ya es undefined.

```suggestion
        const bytes = response?.QueryExecution?.Statistics?.DataScannedInBytes;
```